Source/effects.cpp

```diff
@@ -1011,83 +1011,54 @@ void __fastcall PlaySFX(int psfx)
 	PlaySFX_priv(&sgSFX[v1], 0, 0, 0);
 }
 
-void __fastcall PlaySFX_priv(TSFX *pSFX, char loc, int x, int y)
+void __fastcall PlaySFX_priv(TSFX *pSFX, int loc, int x, int y)

```

Comment:
Is loc a BOOL?

Reply:
Odd, I'm sure I pushed it, I'll push again and make sure that it goes thru

Reply:
There we go, now it's actually fixed